changes in ajaxPost, renaming TextField (because it can represents other tpyes of fields),SplitButtonDropdown

bbodi opened the pull request:
- ajaxPost
- TextField -> InputField
- Bootstrap SplitButtonDropdown

jean79 commented:
TextField -> InputField and ajaxPost are ok.
Though I have committed dropdown just an hour before you :( It is almost the same as yours. Just the name is different.
Would you mind removing it from this pull request?
We will have to probably create Issue prior to each bigger development and assign it to yourself. Then we would avoid working on the same components at the same time, ok?

bbodi commented:
```
Would you mind removing it from this pull request?
```
Ok, no problem
```
We will have to probably create Issue prior to each bigger development and assign it to yourself
```
Totally understand.

bbodi commented:
```
Though I have committed dropdown just an hour before you :( It is almost the same as yours. Just the name is different.
```
I implemented split button dropdown. It's different from simple dropdown, and it's based on your dropdown implementation.  
I created a demo for it on the bootstrap page, please check it.   
Split button dropdown differs from simple dropdown in that it has a separate button and a down-arrow button next to it.
